src/Numerics/BasicMath.php

```diff
@@ -0,0 +1,30 @@
+<?php namespace Moserware\Skills\Numerics;
+
+/**
+ * Basic math functions.
+ * 
+ * @author     Jeff Moser <jeff@moserware.com>
+ * @copyright  2010 Jeff Moser 
+ */
+class BasicMath {
```

Comment:
Note that this has different brace conventions than other files. I myself now prefer this style, but when I wrote this repo, I used one on a new line. I don't really care which one is followed in this repo, just that it's consistent. Perhaps move all to PSR-2 or make on a new line consistent here?